<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateBidPopup</class>
 <widget class="QDialog" name="CreateBidPopup">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Create Bid</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
    background-color: #1E1E1E;
    color: #FFFFFF;
}

QLineEdit, QTextEdit, QPlainTextEdit {
    background-color: #2E2E2E;
    color: #FFFFFF;
}

QPushButton {
    background-color: #404040;
    color: #FFFFFF;
    border: 1px solid #606060;
    border-radius: 10px; 
    padding: 5px 10px;  
}
</string>
  </property>
  <widget class="QLineEdit" name="QuantityLine">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>120</y>
     <width>321</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="RateLine">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>190</y>
     <width>321</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>160</y>
     <width>171</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Rate (USD to RUB)</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>90</y>
     <width>171</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Quantity</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Ok">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>230</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Ok</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>50</y>
     <width>321</width>
     <height>41</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Selling USD for RUB</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Buying RUB for USD</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>30</y>
     <width>171</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Type</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
